NEWS_DEV.txt

```diff
@@ -115,6 +122,9 @@ GPU:
    - Started to use ``float32`` precision for computations that don't support ``float16`` on GPU
 
 New features:
+ - Implemented truncated normal distribution with box-muller transform
+ - Added ``L_op()`` overriding option for ``OpFromGraph``
```

Comment:
where this come from?

Reply:
From here: https://github.com/Theano/Theano/pull/5641/

Reply:
ok